test/helpers/evmScript.js

```diff
@@ -0,0 +1,13 @@
+const abi = require('ethereumjs-abi')
+
+module.exports = {
+    encodeScript: actions => {
```

Comment:
This looks interesting, but I think for the sake of clarity we should make a habit out of documenting our test helpers as well

Reply:
You are very right, fixed!